### ISSUE TITLE
Confirm our Vitest config and unit test setup

### ISSUE THREAD

**Issue by humphd** [DevelopingSpace/starchart#134]
We ran into some bugs today with version issues that triggered odd failures in our Vitest unit test run.  I think we should look at how this is configured and tweak it if necessary.  I've not used Vitest before, so I don't have a lot of advice.  Whoever does this should go through their docs and understand how ours is setup, and make changes if necessary.

Especially since we're going to be using Playwright for most UI tests, we can use this for unit testing in node mostly I think.

**Comment by humphd** [comment]
One thing I know we can remove as part of this is the `utils.test.ts` file in `app/` which we aren't using.  We don't need the `validateEmail()` function in `utils.ts` either.

**Comment by humphd** [comment]
I don't think we need to do more here, it's been working OK?

The only thing I notice is that it's using a fake DOM (happy dom) vs. node environment.  I wonder if we should switch to node?

**Comment by Eakam1007** [comment]
Yea, sorry haven't had the chance to look at this. I am hoping to confirm this over the weekend but I am pretty sure not much needs to be changed here

**Comment by Eakam1007** [comment]
From what I was able to tell, we can definitely switch to node as that is the default environment. We can also update the exclude and include configurations as they include some stuff we don't have. For instance, `postgres-data` could probably be replaced with `mysql-data`.
Also, I noticed that we do not have a script for running coverage, which would be nice to have.
I am planning to make these changes in release 0.7 unless you think something here is unnecessary @humphd
